### PR TITLE
FLHO-345-shooting clubs: Adding the Club Secretary Email form page

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
https://jira.digital.homeoffice.gov.uk/browse/FLHO-345

### CODE REVIEW

File: apps/shooting-clubs/acceptance/features/club-secretary-email.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+) => {
+  I.visitPage(clubSecretaryEmailPage, steps);
+});
+
+Scenario('check there is the correct form elements', (
+  I,
+  clubSecretaryEmailPage
+) => {
+  I.seeElement(clubSecretaryEmailPage['club-secretary-email-id']);
+  I.seeElement(clubSecretaryEmailPage['club-secretary-phone-id']);
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/club-secretary-email.js]: just as an fyi there is a helper method you can use for multiple elements
```
I.seeElements([elements])
```

it also accepts a single element string

File: apps/shooting-clubs/acceptance/features/club-secretary-email.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+});
+
+Scenario('check that an error appears if I submit empty fields', (
+  I,
+  clubSecretaryEmailPage
+) => {
+  I.submitForm();
+  I.seeErrors(clubSecretaryEmailPage['club-secretary-email-id']);
+  I.seeErrors(clubSecretaryEmailPage['club-secretary-phone-id']);
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/features/club-secretary-email.js]: `seeErrors` accepts a string or array - you can call it once

File: apps/shooting-clubs/acceptance/pages/club-secretary-email.js [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+let I;
 module.exports = {
-  url: 'club-secretary-email'
+  _init() {
+    I = require('so-acceptance/steps.js')();
+  },
+
+  url: 'club-secretary-email',
+  'club-secretary-email-id': '#club-secretary-email',
+  'club-secretary-phone-id': '#club-secretary-phone',
```
Comment (joefitter) [inline apps/shooting-clubs/acceptance/pages/club-secretary-email.js]: if you put both of these in a separate fields object you can reference them easier in your tests with something like:
```
I.seeElements(Object.keys(page.fields));
```

File: apps/shooting-clubs/index.js [hunk 4]
```diff
@@ -114,6 +114,10 @@ module.exports = {
       }
     },
     '/club-secretary-email': {
+        fields: [
```
Comment (bendiggle) [inline apps/shooting-clubs/index.js]: indentation oops

File: apps/shooting-clubs/acceptance/pages/club-secretary-email.js (hunk 3 above)
Comment (bendiggle) [inline apps/shooting-clubs/acceptance/pages/club-secretary-email.js]: ^ Show off

File: apps/shooting-clubs/acceptance/features/club-secretary-email.js (hunk 1 above)
Comment (sulthan-ahmed) [inline apps/shooting-clubs/acceptance/features/club-secretary-email.js]: 👍

File: apps/shooting-clubs/acceptance/features/club-secretary-email.js (hunk 2 above)
Comment (sulthan-ahmed) [inline apps/shooting-clubs/acceptance/features/club-secretary-email.js]: nice

File: apps/shooting-clubs/acceptance/pages/club-secretary-email.js (hunk 3 above)
Comment (joefitter) [inline apps/shooting-clubs/acceptance/pages/club-secretary-email.js]: 💋

File: apps/shooting-clubs/index.js (hunk 4 above)
Comment (joefitter) [inline apps/shooting-clubs/index.js]: https://atom.io/

File: apps/shooting-clubs/acceptance/pages/club-secretary-email.js (hunk 3 above)
Comment (sulthan-ahmed) [inline apps/shooting-clubs/acceptance/pages/club-secretary-email.js]: hmm, need to work out what you mean :thinking:

File: apps/shooting-clubs/index.js (hunk 4 above)
Comment (sulthan-ahmed) [inline apps/shooting-clubs/index.js]: lol, but intellij is so good at debugging and resolving conflicts :|

**Pull request merged**